### PR TITLE
[SAIC-186] Make log config path configurable

### PR DESCRIPTION
**Pull request by vladiskuz**
This pull request contain 2 commits.  Explanation for each commit you can find in commit messages.

### CODE REVIEW

**Comment by MirantisWorkloadMobility** [comment]
Consider resolving pylint issues: \
``` \
************* Module cloudferrylib.os.actions.check_config_quota_neutron
cloudferrylib/os/actions/check_config_quota_neutron.py:29: [W0613(unused-argument), CheckConfigQuotaNeutron.run] Unused argument 'info'
************* Module cloud.grouping
cloud/grouping.py:17: [W0403(relative-import), ] Relative import 'cloud', should be 'cloud.cloud'
cloud/grouping.py:102: [W0612(unused-variable), Grouping._group_nested_network] Unused variable 'network_name'
cloud/grouping.py:121: [W0141(bad-builtin), Grouping._make_users_group] Used builtin function 'filter'. Using a list comprehension can be clearer.
cloud/grouping.py:151: [W0106(expression-not-assigned), Grouping._walk_user_defined] Expression "[value.remove(i) for i in same if i in value]" is assigned to nothing
cloud/grouping.py:145: [W0612(unused-variable), Grouping._walk_user_defined] Unused variable 'key'
************* Module cloudferrylib.os.actions.check_ssh
cloudferrylib/os/actions/check_ssh.py:40: [W0141(bad-builtin), CheckSSH.run] Used builtin function 'map'. Using a list comprehension can be clearer.
cloudferrylib/os/actions/check_ssh.py:41: [W0141(bad-builtin), CheckSSH.run] Used builtin function 'filter'. Using a list comprehension can be clearer.
cloudferrylib/os/actions/check_ssh.py:41: [W0110(deprecated-lambda), CheckSSH.run] map/filter on lambda could be replaced by comprehension
cloudferrylib/os/actions/check_ssh.py:29: [W0613(unused-argument), CheckSSH.run] Unused argument 'info'
cloudferrylib/os/actions/check_ssh.py:60: [W0703(broad-except), CheckSSH.check_access] Catching too general exception Exception
************* Module cloudferrylib.os.actions.check_users
cloudferrylib/os/actions/check_users.py:48: [W0141(bad-builtin), CheckUsersAvailabilityOnSrcAndDst.run] Used builtin function 'map'. Using a list comprehension can be clearer.
cloudferrylib/os/actions/check_users.py:49: [W0141(bad-builtin), CheckUsersAvailabilityOnSrcAndDst.run] Used builtin function 'map'. Using a list comprehension can be clearer.
************* Module cloudferrylib.os.actions.convert_file
cloudferrylib/os/actions/convert_file.py:17: [W0612(unused-variable), ConvertFile.run] Unused variable 'instance'
************* Module cloudferrylib.os.actions.convert_volume_to_image
cloudferrylib/os/actions/convert_volume_to_image.py:38: [W0102(dangerous-default-value), ConvertVolumeToImage.run] Dangerous default value {} as argument
cloudferrylib/os/actions/convert_volume_to_image.py:54: [W0612(unused-variable), ConvertVolumeToImage.run] Unused variable 'resp'
cloudferrylib/os/actions/convert_volume_to_image.py:48: [W0612(unused-variable), ConvertVolumeToImage.run] Unused variable 'volume_id'
cloudferrylib/os/actions/convert_volume_to_image.py:40: [W0201(attribute-defined-outside-init), ConvertVolumeToImage.run] Attribute 'container_format' defined outside __init__
cloudferrylib/os/actions/convert_volume_to_image.py:39: [W0201(attribute-defined-outside-init), ConvertVolumeToImage.run] Attribute 'disk_format' defined outside __init__
************* Module cloudferrylib.os.actions.failure_act
cloudferrylib/os/actions/failure_act.py:25: [W0613(unused-argument), DivisionByZero.run] Unused argument 'args'
************* Module cloudferrylib.os.actions.image_snapshot
cloudferrylib/os/actions/image_snapshot.py:41: [W0613(unused-argument), ImageSnapshot.run] Unused argument 'args'
cloudferrylib/os/actions/image_snapshot.py:47: [W0613(unused-argument), ImageRollback.run] Unused argument 'args'
************* Module cloudferrylib.os.actions.filter_similar_vms_from_dst
cloudferrylib/os/actions/filter_similar_vms_from_dst.py:66: [W0201(attribute-defined-outside-init), FilterSimilarVMsFromDST.find_similar_instances] Attribute 'dst_instances' defined outside __init__
cloudferrylib/os/actions/filter_similar_vms_from_dst.py:42: [W0201(attribute-defined-outside-init), FilterSimilarVMsFromDST.run] Attribute 'tenant_id_to_new_id' defined outside __init__
cloudferrylib/os/actions/filter_similar_vms_from_dst.py:68: [W0201(attribute-defined-outside-init), FilterSimilarVMsFromDST.find_similar_instances] Attribute 'similar_isntances' defined outside __init__
cloudferrylib/os/actions/filter_similar_vms_from_dst.py:41: [W0201(attribute-defined-outside-init), FilterSimilarVMsFromDST.run] Attribute 'src_instances' defined outside __init__
cloudferrylib/os/actions/filter_similar_vms_from_dst.py:69: [W0201(attribute-defined-outside-init), FilterSimilarVMsFromDST.find_similar_instances] Attribute 'conflict_instances' defined outside __init__
************* Module cloudferrylib.os.actions.prepare_networks
cloudferrylib/os/actions/prepare_networks.py:151: [W0612(unused-variable), CreatePortsForVRRP.run] Unused variable 'id_inst'
************* Module cloudferrylib.os.actions.recreate_boot_image
cloudferrylib/os/actions/recreate_boot_image.py:39: [W0102(dangerous-default-value), ReCreateBootImage.run] Dangerous default value {} as argument
cloudferrylib/os/actions/recreate_boot_image.py:39: [W0102(dangerous-default-value), ReCreateBootImage.run] Dangerous default value {} as argument
************* Module cloudferrylib.os.actions.live_migration
cloudferrylib/os/actions/live_migration.py:180: [W0613(unused-argument), RestoreOriginalInstancesTableSchema.run] Unused argument 'info'
cloudferrylib/os/actions/live_migration.py:199: [W0613(unused-argument), CheckQemuVersion.run] Unused argument 'info'
************* Module cloudferrylib.os.actions.snapshots
cloudferrylib/os/actions/snapshots.py:45: [W0613(unused-argument), MysqlDump.run] Unused argument 'args'
cloudferrylib/os/actions/snapshots.py:87: [W0613(unused-argument), MysqlRestore.run] Unused argument 'args'
************* Module cloudferrylib.os.actions.utils
cloudferrylib/os/actions/utils.py:42: [E1102(not-callable), transfer_file_to_file] utils.up_ssh_tunnel is not callable
************* Module cloudferrylib.os.actions.transport_compute_resources
cloudferrylib/os/actions/transport_compute_resources.py:85: [W0613(unused-argument), TransportKeyPairs.__init__] Unused argument 'kwargs'
cloudferrylib/os/actions/transport_compute_resources.py:122: [W0613(unused-argument), SetKeyPairsForInstances.__init__] Unused argument 'kwargs'
************* Module cloudferrylib.os.actions.transport_ephemeral
cloudferrylib/os/actions/transport_ephemeral.py:175: [W0612(unused-variable), TransportEphemeral.copy_ephemeral_iscsi_to_ceph] Unused variable 'inst_id'
************* Module cloudferrylib.os.actions.vm_snapshot
cloudferrylib/os/actions/vm_snapshot.py:47: [W0613(unused-argument), VmSnapshot.run] Unused argument 'args'
cloudferrylib/os/actions/vm_snapshot.py:58: [W0613(unused-argument), VmRestore.run] Unused argument 'args'
************* Module cloudferrylib.os.compute.keypairs
cloudferrylib/os/compute/keypairs.py:66: [W0141(bad-builtin), KeyPair.to_dict] Used builtin function 'filter'. Using a list comprehension can be clearer.
cloudferrylib/os/compute/keypairs.py:66: [W0110(deprecated-lambda), KeyPair.to_dict] map/filter on lambda could be replaced by comprehension
cloudferrylib/os/compute/keypairs.py:85: [W0141(bad-builtin), DBBroker.get_all_keypairs] Used builtin function 'map'. Using a list comprehension can be clearer.
************* Module cloudferrylib.os.compute.cold_evacuate
cloudferrylib/os/compute/cold_evacuate.py:177: [I0011(locally-disabled), ] Locally disabling broad-except (W0703)
cloudferrylib/os/compute/cold_evacuate.py:198: [I0011(locally-disabled), ] Locally disabling broad-except (W0703)
cloudferrylib/os/compute/cold_evacuate.py:274: [E1101(no-member), install_ssh_keys] Instance of 'ValueError' has no 'replace' member
cloudferrylib/os/compute/cold_evacuate.py:275: [E1101(no-member), install_ssh_keys] Instance of 'ValueError' has no 'replace' member
************* Module cloudferrylib.os.compute.libvirt
cloudferrylib/os/compute/libvirt.py:80: [W0702(bare-except), DestNovaInstanceDestroyer.undo] No exception type(s) specified
cloudferrylib/os/compute/libvirt.py:197: [W0141(bad-builtin), LibvirtXml.__init__] Used builtin function 'map'. Using a list comprehension can be clearer.
************* Module cloudferrylib.os.compute.nova_compute
cloudferrylib/os/compute/nova_compute.py:20: [E0611(no-name-in-module), ] No name 'client' in module 'novaclient.v1_1'
cloudferrylib/os/compute/nova_compute.py:774: [E0602(undefined-variable), NovaCompute.change_status.<lambda>.<lambda>] Undefined variable 'status'
************* Module cloudferrylib.os.network.nova_network
cloudferrylib/os/network/nova_network.py:20: [E0611(no-name-in-module), ] No name 'client' in module 'novaclient.v1_1'
cloudferrylib/os/network/nova_network.py:58: [W0221(arguments-differ), NovaNetwork.deploy] Arguments number differs from overridden 'deploy' method
cloudferrylib/os/network/nova_network.py:61: [W0221(arguments-differ), NovaNetwork.get_security_groups] Arguments number differs from overridden 'get_security_groups' method
************* Module cloudferrylib.os.image.glance_image
cloudferrylib/os/image/glance_image.py:225: [W0212(protected-access), GlanceImage.get_ref_image] Access to a protected member _resp of a client class
cloudferrylib/os/image/glance_image.py:305: [W0613(unused-argument), GlanceImage.read_info] Unused argument 'kwargs'
cloudferrylib/os/image/glance_image.py:369: [W0221(arguments-differ), GlanceImage.deploy] Arguments number differs from overridden 'deploy' method
************* Module cloudferrylib.os.storage.cinder_netapp
cloudferrylib/os/storage/cinder_netapp.py:45: [W0221(arguments-differ), CinderNetApp.fix_entries] Arguments number differs from overridden 'fix_entries' method
cloudferrylib/os/storage/cinder_netapp.py:46: [E1121(too-many-function-args), CinderNetApp.fix_entries] Too many positional arguments for method call
************* Module cloudferrylib.os.storage.cinder_database
cloudferrylib/os/storage/cinder_database.py:225: [W0125(using-constant-test), CinderStorage._filter_quotas_list] Using a conditional statement with a constant value
************* Module cloudferrylib.os.storage.cinder_storage
cloudferrylib/os/storage/cinder_storage.py:97: [W0221(arguments-differ), CinderStorage.deploy] Arguments number differs from overridden 'deploy' method
cloudferrylib/os/storage/cinder_storage.py:254: [W0613(unused-argument), CinderStorage.convert_snapshot] Unused argument 'cloud'
************* Module cloudferrylib.os.identity.keystone
cloudferrylib/os/identity/keystone.py:194: [W0221(arguments-differ), KeystoneIdentity.deploy] Arguments number differs from overridden 'deploy' method
************* Module cloudferrylib.utils.drivers.ssh_ceph_to_ceph
cloudferrylib/utils/drivers/ssh_ceph_to_ceph.py:26: [W0221(arguments-differ), SSHCephToCeph.transfer] Arguments number differs from overridden 'transfer' method
cloudferrylib/utils/drivers/ssh_ceph_to_ceph.py:31: [E1129(not-context-manager), SSHCephToCeph.transfer] Context manager 'tuple' doesn't implement __enter__ and __exit__.
************* Module cloudferrylib.utils.drivers.ssh_ceph_to_file
cloudferrylib/utils/drivers/ssh_ceph_to_file.py:28: [E1102(not-callable), SSHCephToFile.transfer] utils.up_ssh_tunnel is not callable
************* Module cloudferrylib.scheduler.scheduler
cloudferrylib/scheduler/scheduler.py:25: [W0403(relative-import), ] Relative import 'cursor', should be 'cloudferrylib.scheduler.cursor'
cloudferrylib/scheduler/scheduler.py:26: [W0403(relative-import), ] Relative import 'signal_handler', should be 'cloudferrylib.scheduler.signal_handler'
cloudferrylib/scheduler/scheduler.py:27: [W0403(relative-import), ] Relative import 'task', should be 'cloudferrylib.scheduler.task'
cloudferrylib/scheduler/scheduler.py:28: [W0403(relative-import), ] Relative import 'thread_tasks', should be 'cloudferrylib.scheduler.thread_tasks'
cloudferrylib/scheduler/scheduler.py:58: [W0613(unused-argument), BaseScheduler.event_error_task] Unused argument 'task'
cloudferrylib/scheduler/scheduler.py:58: [W0613(unused-argument), BaseScheduler.event_error_task] Unused argument 'e'
cloudferrylib/scheduler/scheduler.py:76: [W0703(broad-except), BaseScheduler.process_chain] Catching too general exception Exception
cloudferrylib/scheduler/scheduler.py:85: [W0201(attribute-defined-outside-init), BaseScheduler.process_chain] Attribute 'exception' defined outside __init__
cloudferrylib/scheduler/scheduler.py:108: [W0201(attribute-defined-outside-init), BaseScheduler.addCursor] Attribute 'cursor' defined outside __init__
************* Module cloudferrylib.utils.drivers.ssh_file_to_ceph
cloudferrylib/utils/drivers/ssh_file_to_ceph.py:32: [E1129(not-context-manager), SSHFileToCeph.transfer] Context manager 'tuple' doesn't implement __enter__ and __exit__.
************* Module cloudferrylib.utils.file_like_proxy
cloudferrylib/utils/file_like_proxy.py:59: [W0141(bad-builtin), FileLikeProxy._parse_speed_limit] Used builtin function 'filter'. Using a list comprehension can be clearer.
cloudferrylib/utils/file_like_proxy.py:74: [E0202(method-hidden), FileLikeProxy.read] An attribute defined in cloudferrylib.utils.file_like_proxy line 43 hides this method
************* Module cloudferrylib.utils.drivers.ssh_file_to_file
cloudferrylib/utils/drivers/ssh_file_to_file.py:36: [E1102(not-callable), SSHFileToFile.transfer] utils.up_ssh_tunnel is not callable
cloudferrylib/utils/drivers/ssh_file_to_file.py:80: [E1129(not-context-manager), SSHFileToFile.transfer_direct] Context manager 'tuple' doesn't implement __enter__ and __exit__.
************* Module cloudferrylib.utils.proxy_client
cloudferrylib/utils/proxy_client.py:78: [I0011(locally-disabled), ] Locally disabling broad-except (W0703)
************* Module cloudferrylib.os.network.neutron
cloudferrylib/os/network/neutron.py:179: [W0221(arguments-differ), NeutronNetwork.deploy] Arguments number differs from overridden 'deploy' method
cloudferrylib/os/network/neutron.py:284: [W0221(arguments-differ), NeutronNetwork.create_port] Arguments number differs from overridden 'create_port' method
cloudferrylib/os/network/neutron.py:816: [W0221(arguments-differ), NeutronNetwork.get_security_groups] Arguments number differs from overridden 'get_security_groups' method
************* Module cloudferrylib.utils.ssh_util
cloudferrylib/utils/ssh_util.py:18: [W0403(relative-import), ] Relative import 'cmd_cfg', should be 'cloudferrylib.utils.cmd_cfg'
cloudferrylib/utils/ssh_util.py:19: [W0403(relative-import), ] Relative import 'utils', should be 'cloudferrylib.utils.utils'
************* Module condensation.action
condensation/action.py:77: [E1120(no-value-for-parameter), Actions.dump_actions] No value for argument 'connection' in function call
condensation/action.py:87: [W0141(bad-builtin), Actions.add_migration_action] Used builtin function 'map'. Using a list comprehension can be clearer.
condensation/action.py:106: [W0141(bad-builtin), Actions.add_condensation_action] Used builtin function 'map'. Using a list comprehension can be clearer.
condensation/action.py:113: [E1120(no-value-for-parameter), get_freed_nodes] No value for argument 'connection' in function call
************* Module cloudferrylib.scheduler.scenario
cloudferrylib/scheduler/scenario.py:112: [W1401(anomalous-backslash-in-string), ] Anomalous backslash in string: '\.'. String constant might be missing an r prefix.
cloudferrylib/scheduler/scenario.py:37: [W0102(dangerous-default-value), Scenario.init_tasks] Dangerous default value {} as argument
cloudferrylib/scheduler/scenario.py:111: [W0141(bad-builtin), Scenario.get_actions] Used builtin function 'filter'. Using a list comprehension can be clearer.
cloudferrylib/scheduler/scenario.py:111: [W0110(deprecated-lambda), Scenario.get_actions] map/filter on lambda could be replaced by comprehension
cloudferrylib/scheduler/scenario.py:114: [W0141(bad-builtin), Scenario.get_actions] Used builtin function 'map'. Using a list comprehension can be clearer.
cloudferrylib/scheduler/scenario.py:114: [W0110(deprecated-lambda), Scenario.get_actions] map/filter on lambda could be replaced by comprehension
cloudferrylib/scheduler/scenario.py:54: [W0201(attribute-defined-outside-init), Scenario.init_tasks] Attribute 'tasks' defined outside __init__
cloudferrylib/scheduler/scenario.py:61: [W0201(attribute-defined-outside-init), Scenario.load_scenario] Attribute 'namespace' defined outside __init__
cloudferrylib/scheduler/scenario.py:66: [W0201(attribute-defined-outside-init), Scenario.load_scenario] Attribute 'preparation' defined outside __init__
cloudferrylib/scheduler/scenario.py:63: [W0201(attribute-defined-outside-init), Scenario.load_scenario] Attribute 'migration' defined outside __init__
cloudferrylib/scheduler/scenario.py:69: [W0201(attribute-defined-outside-init), Scenario.load_scenario] Attribute 'rollback' defined outside __init__
************* Module condensation.scripts.nova_collector
condensation/scripts/nova_collector.py:19: [W0611(unused-import), ] Unused utils imported from cloudferrylib.utils
************* Module data_storage
data_storage.py:89: [W0621(redefined-outer-name), delete_batch] Redefining name 'keys' from outer scope (line 97)
************* Module dry_run.tasks.simple
dry_run/tasks/simple.py:12: [E1003(bad-super-call), AddNumbers.__init__] Bad first argument 'Task' given to super()
dry_run/tasks/simple.py:16: [W0613(unused-argument), AddNumbers.run] Unused argument 'args'
dry_run/tasks/simple.py:24: [E1003(bad-super-call), DivideNumbers.__init__] Bad first argument 'Task' given to super()
dry_run/tasks/simple.py:28: [W0613(unused-argument), DivideNumbers.run] Unused argument 'args'
dry_run/tasks/simple.py:36: [E1003(bad-super-call), MultiplyNumbers.__init__] Bad first argument 'Task' given to super()
dry_run/tasks/simple.py:40: [W0613(unused-argument), MultiplyNumbers.run] Unused argument 'args'
dry_run/tasks/simple.py:49: [W0104(pointless-statement), DivisionByZero.run] Statement seems to have no effect
dry_run/tasks/simple.py:47: [W0613(unused-argument), DivisionByZero.run] Unused argument 'args'
************* Module condensation.cloud
condensation/cloud.py:193: [W0612(unused-variable), Cloud.fil_node] Unused variable 'counter'
************* Module evacuation.actions.evacuate_vms
evacuation/actions/evacuate_vms.py:41: [E1120(no-value-for-parameter), Evacuate.run] No value for argument 'connection' in function call
************* Module make_filters.make_filters
make_filters/make_filters.py:32: [E1120(no-value-for-parameter), delete_relations] No value for argument 'connection' in function call
make_filters/make_filters.py:33: [E1120(no-value-for-parameter), delete_relations] No value for argument 'connection' in function call
make_filters/make_filters.py:48: [E1120(no-value-for-parameter), make] No value for argument 'connection' in function call
make_filters/make_filters.py:56: [E1120(no-value-for-parameter), make] No value for argument 'connection' in function call
************* Module cloudferrylib.utils.utils
cloudferrylib/utils/utils.py:130: [W0201(attribute-defined-outside-init), Postman.__enter__] Attribute 'server' defined outside __init__
cloudferrylib/utils/utils.py:195: [W0201(attribute-defined-outside-init), ForwardAgent.__enter__] Attribute 'pid' defined outside __init__
cloudferrylib/utils/utils.py:196: [W0201(attribute-defined-outside-init), ForwardAgent.__enter__] Attribute 'ssh_auth_sock' defined outside __init__
cloudferrylib/utils/utils.py:254: [W0201(attribute-defined-outside-init), UpSshTunnelClass.__enter__] Attribute 'port' defined outside __init__
************* Module tests.cloudferrylib.os.actions.test_converter_volume_to_image
tests/cloudferrylib/os/actions/test_converter_volume_to_image.py:25: [E1002(super-on-old-class), ConverterVolumeToImageTest.setUp] Use of super on an old style class
************* Module tests.cloudferrylib.os.actions.test_get_info_images
tests/cloudferrylib/os/actions/test_get_info_images.py:25: [E1002(super-on-old-class), GetInfoImagesTestCase.setUp] Use of super on an old style class
************* Module tests.cloud.test_grouping
tests/cloud/test_grouping.py:40: [E1002(super-on-old-class), GroupingTestCase.setUp] Use of super on an old style class
tests/cloud/test_grouping.py:96: [E1002(super-on-old-class), GroupingTestCase.tearDown] Use of super on an old style class
************* Module tests.cloudferrylib.os.actions.test_copy_g2g
tests/cloudferrylib/os/actions/test_copy_g2g.py:25: [E1002(super-on-old-class), CopyFromGlanceToGlanceTestCase.setUp] Use of super on an old style class
************* Module tests.cloudferrylib.os.compute.test_server_groups
tests/cloudferrylib/os/compute/test_server_groups.py:18: [E0611(no-name-in-module), ] No name 'client' in module 'novaclient.v1_1'
tests/cloudferrylib/os/compute/test_server_groups.py:45: [E1002(super-on-old-class), ServerGroupTestCase.setUp] Use of super on an old style class
tests/cloudferrylib/os/compute/test_server_groups.py:109: [W0212(protected-access), ServerGroupTestCase.test_deploy_server_groups_already_exists] Access to a protected member _delete_server_group of a client class
tests/cloudferrylib/os/compute/test_server_groups.py:110: [W0212(protected-access), ServerGroupTestCase.test_deploy_server_groups_already_exists] Access to a protected member _deploy_server_group of a client class
tests/cloudferrylib/os/compute/test_server_groups.py:113: [W0212(protected-access), ServerGroupTestCase.test_deploy_server_groups_already_exists] Access to a protected member _delete_server_group of a client class
tests/cloudferrylib/os/compute/test_server_groups.py:114: [W0212(protected-access), ServerGroupTestCase.test_deploy_server_groups_already_exists] Access to a protected member _deploy_server_group of a client class
tests/cloudferrylib/os/compute/test_server_groups.py:129: [W0212(protected-access), ServerGroupTestCase.test_deploy_server_groups_already_exists_doesnt_match] Access to a protected member _delete_server_group of a client class
tests/cloudferrylib/os/compute/test_server_groups.py:130: [W0212(protected-access), ServerGroupTestCase.test_deploy_server_groups_already_exists_doesnt_match] Access to a protected member _deploy_server_group of a client class
tests/cloudferrylib/os/compute/test_server_groups.py:133: [W0212(protected-access), ServerGroupTestCase.test_deploy_server_groups_already_exists_doesnt_match] Access to a protected member _delete_server_group of a client class
tests/cloudferrylib/os/compute/test_server_groups.py:140: [W0212(protected-access), ServerGroupTestCase.test_deploy_server_groups_already_exists_doesnt_match] Access to a protected member _deploy_server_group of a client class
************* Module tests.cloudferrylib.os.compute.test_cold_evacuate
tests/cloudferrylib/os/compute/test_cold_evacuate.py:14: [I0011(locally-disabled), ] Locally disabling no-member (E1101)
************* Module tests.cloudferrylib.os.image.test_glance_image
tests/cloudferrylib/os/image/test_glance_image.py:46: [E1002(super-on-old-class), GlanceImageTestCase.setUp] Use of super on an old style class
tests/cloudferrylib/os/image/test_glance_image.py:50: [W0212(protected-access), GlanceImageTestCase.setUp] Access to a protected member _resp of a client class
************* Module tests.cloudferrylib.os.compute.test_nova
tests/cloudferrylib/os/compute/test_nova.py:19: [E0611(no-name-in-module), ] No name 'client' in module 'novaclient.v1_1'
tests/cloudferrylib/os/compute/test_nova.py:44: [E1002(super-on-old-class), NovaComputeTestCase.setUp] Use of super on an old style class
tests/cloudferrylib/os/compute/test_nova.py:338: [W0212(protected-access), FlavorDeploymentTestCase.test_flavor_is_updated_with_destination_id] Access to a protected member _create_flavor_if_not_exists of a client class
tests/cloudferrylib/os/compute/test_nova.py:339: [W0212(protected-access), FlavorDeploymentTestCase.test_flavor_is_updated_with_destination_id] Access to a protected member _create_flavor_if_not_exists of a client class
tests/cloudferrylib/os/compute/test_nova.py:365: [W0212(protected-access), FlavorDeploymentTestCase.test_flavor_is_updated_with_destination_id] Access to a protected member _deploy_flavors of a client class
tests/cloudferrylib/os/compute/test_nova.py:392: [W0212(protected-access), FlavorDeploymentTestCase.test_flavor_is_not_created_if_already_exists_on_dest] Access to a protected member _create_flavor_if_not_exists of a client class
tests/cloudferrylib/os/compute/test_nova.py:395: [W0212(protected-access), FlavorDeploymentTestCase.test_flavor_is_not_created_if_already_exists_on_dest] Access to a protected member _deploy_flavors of a client class
tests/cloudferrylib/os/compute/test_nova.py:397: [W0212(protected-access), FlavorDeploymentTestCase.test_flavor_is_not_created_if_already_exists_on_dest] Access to a protected member _create_flavor_if_not_exists of a client class
tests/cloudferrylib/os/compute/test_nova.py:416: [W0212(protected-access), FlavorDeploymentTestCase.test_access_not_updated_for_public_flavors] Access to a protected member _create_flavor_if_not_exists of a client class
tests/cloudferrylib/os/compute/test_nova.py:417: [W0212(protected-access), FlavorDeploymentTestCase.test_access_not_updated_for_public_flavors] Access to a protected member _add_flavor_access_for_tenants of a client class
tests/cloudferrylib/os/compute/test_nova.py:421: [W0212(protected-access), FlavorDeploymentTestCase.test_access_not_updated_for_public_flavors] Access to a protected member _deploy_flavors of a client class
tests/cloudferrylib/os/compute/test_nova.py:423: [W0212(protected-access), FlavorDeploymentTestCase.test_access_not_updated_for_public_flavors] Access to a protected member _add_flavor_access_for_tenants of a client class
tests/cloudferrylib/os/compute/test_nova.py:442: [W0212(protected-access), FlavorDeploymentTestCase.test_access_list_is_updated_for_non_public_flavors] Access to a protected member _create_flavor_if_not_exists of a client class
tests/cloudferrylib/os/compute/test_nova.py:443: [W0212(protected-access), FlavorDeploymentTestCase.test_access_list_is_updated_for_non_public_flavors] Access to a protected member _add_flavor_access_for_tenants of a client class
tests/cloudferrylib/os/compute/test_nova.py:447: [W0212(protected-access), FlavorDeploymentTestCase.test_access_list_is_updated_for_non_public_flavors] Access to a protected member _deploy_flavors of a client class
tests/cloudferrylib/os/compute/test_nova.py:449: [W0212(protected-access), FlavorDeploymentTestCase.test_access_list_is_updated_for_non_public_flavors] Access to a protected member _add_flavor_access_for_tenants of a client class
************* Module tests.cloudferrylib.os.network.test_nova_network
tests/cloudferrylib/os/network/test_nova_network.py:18: [E0611(no-name-in-module), ] No name 'client' in module 'novaclient.v1_1'
tests/cloudferrylib/os/network/test_nova_network.py:37: [E1002(super-on-old-class), TestNovaNetwork.setUp] Use of super on an old style class
************* Module tests.cloudferrylib.os.identity.test_keystone
tests/cloudferrylib/os/identity/test_keystone.py:49: [E1002(super-on-old-class), KeystoneIdentityTestCase.setUp] Use of super on an old style class
tests/cloudferrylib/os/identity/test_keystone.py:214: [W0212(protected-access), KeystoneIdentityTestCase.test_read_info] Access to a protected member _get_user_roles_cached of a client class
tests/cloudferrylib/os/identity/test_keystone.py:215: [W0212(protected-access), KeystoneIdentityTestCase.test_read_info] Access to a protected member _get_user_roles_cached of a client class
tests/cloudferrylib/os/identity/test_keystone.py:237: [W0613(unused-argument), KeystoneIdentityTestCase.test_deploy.tenant_create] Unused argument 'kwargs'
tests/cloudferrylib/os/identity/test_keystone.py:242: [W0613(unused-argument), KeystoneIdentityTestCase.test_deploy.user_create] Unused argument 'kwars'
tests/cloudferrylib/os/identity/test_keystone.py:247: [W0613(unused-argument), KeystoneIdentityTestCase.test_deploy.roles_create] Unused argument 'role_name'
tests/cloudferrylib/os/identity/test_keystone.py:447: [W0212(protected-access), KeystoneClientTestCase.test_adds_region_if_set_in_config] Access to a protected member _get_client_by_creds of a client class
tests/cloudferrylib/os/identity/test_keystone.py:479: [W0212(protected-access), KeystoneClientTestCase.test_does_not_add_region_if_not_set_in_config] Access to a protected member _get_client_by_creds of a client class
************* Module tests.cloudferrylib.os.storage.test_cinder_netapp
tests/cloudferrylib/os/storage/test_cinder_netapp.py:48: [E1002(super-on-old-class), CinderNetAppTestCase.setUp] Use of super on an old style class
************* Module tests.cloudferrylib.utils.test_runner
tests/cloudferrylib/utils/test_runner.py:41: [W0703(broad-except), RemoteRunnerTestCase.test_errors_are_suppressed_for_run_ignoring_errors] Catching too general exception Exception
************* Module tests.cloudferrylib.utils.test_utils
tests/cloudferrylib/utils/test_utils.py:33: [W0212(protected-access), ForwardAgentTestCase.test_agent_is_already_run_w_keys] Access to a protected member _agent_already_running of a client class
tests/cloudferrylib/utils/test_utils.py:43: [W0212(protected-access), ForwardAgentTestCase.test_agent_is_not_run] Access to a protected member _agent_already_running of a client class
tests/cloudferrylib/utils/test_utils.py:53: [W0212(protected-access), ForwardAgentTestCase.test_agent_is_already_run_w_another_key] Access to a protected member _agent_already_running of a client class
************* Module tests.cloudferrylib.os.network.test_neutron
tests/cloudferrylib/os/network/test_neutron.py:48: [E1002(super-on-old-class), NeutronTestCase.setUp] Use of super on an old style class
************* Module tests.cloudferrylib.os.storage.test_cinder_storage
tests/cloudferrylib/os/storage/test_cinder_storage.py:46: [E1002(super-on-old-class), CinderStorageTestCase.setUp] Use of super on an old style class
tests/cloudferrylib/os/storage/test_cinder_storage.py:232: [W0703(broad-except), CinderStorageTestCase.test_get_volume_path_iscsi_error] Catching too general exception Exception

**Comment by MirantisWorkloadMobility** [comment]
PEP8 check success, Unittests success

File: fabfile.py [hunk 1]
```diff
@@ -200,16 +188,18 @@ def create_filters(name_config=None, filter_folder=DEFAULT_FILTERS_FILES,
...
 
-def load_config(name_config):
+def init_config_and_log(name_config):
     cfglib.collector_configs_plugins()
     cfglib.init_config(name_config)
     utils.init_singletones(cfglib.CONF)
     if cfglib.CONF.migrate.hide_ssl_warnings:
         warnings.simplefilter("ignore")
+    log.setup(cfglib.CONF, 'CloudFerry')
+#    import pdb; pdb.set_trace()
```
Comment (abochkarev) [inline fabfile.py]: Please remove it.

File: configs/logging_config.ini [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+keys=simple,color
+
+[logger_root]
+level=INFO
+handlers=console,info,error
+
+[logger_paramiko.transport]
+handlers=console, info, error
+qualname=paramiko.transport
+
```
Comment (abochkarev) [inline configs/logging_config.ini]: Please add logger for glanceclient.common.http

**Comment by roman-verchikov** [comment]
Continued in https://github.com/MirantisWorkloadMobility/CloudFerry/pull/716

**Pull request closed without merging**